charts/llmisvc-resources/templates/config-llm-decode-worker-data-parallel.yaml

```diff
@@ -52,66 +62,39 @@ spec:
-      - image: ghcr.io/llm-d/llm-d:v0.2.0
+      - image: ghcr.io/llm-d/llm-d-dev:v0.2.2
         imagePullPolicy: IfNotPresent
         name: main
         ports:
           - containerPort: 8001
             protocol: TCP
-        stdin: true
-        tty: true
         command:
           - "/bin/sh"
           - "-c"
         args:
           - |-
-
-            START_RANK=$(( ${LWS_WORKER_INDEX:-0} * {{ or .Spec.Parallelism.DataLocal 1 }} ))
-            if [ "${LWS_WORKER_INDEX:-0}" -eq 0 ]; then
-              #################
-              # Leader-only launch
-              #################
-              vllm serve \
-                {{ .Spec.Model.Name }} \
-                --port 8001 \
-                --api-server-count 4 \
-                --disable-log-requests \
-                {{- if .Spec.Parallelism.Expert -}}--enable-expert-parallel \{{- end }}
-                {{- if .Spec.Parallelism.Tensor -}}--tensor-parallel-size {{ .Spec.Parallelism.Tensor }} \{{- end }}
-                --data-parallel-size {{ or .Spec.Parallelism.Data 1 }} \
-                --data-parallel-size-local {{ or .Spec.Parallelism.DataLocal 1 }} \
-                --data-parallel-address $(LWS_LEADER_ADDRESS) \
-                --data-parallel-rpc-port {{ if .Spec.Parallelism.DataRPCPort }}{{ .Spec.Parallelism.DataRPCPort }}{{ else }}5555{{- end }} \
-                --data-parallel-start-rank $START_RANK \
-                --trust-remote-code \
-                --enable-ssl-refresh \
-                --ssl-certfile \
-                /etc/ssl/certs/tls.crt \
-                --ssl-keyfile \
-                /etc/ssl/certs/tls.key
-            else
-              #################
-              # Worker-only launch
-              #################
-              vllm serve \
-                {{ .Spec.Model.Name }} \
-                --port 8001 \
-                --disable-log-requests \
-                {{- if .Spec.Parallelism.Expert }}--enable-expert-parallel \{{- end }}
-                {{- if .Spec.Parallelism.Tensor }}--tensor-parallel-size {{ .Spec.Parallelism.Tensor }} \{{- end }}
-                --data-parallel-size {{ or .Spec.Parallelism.Data 1 }} \
-                --data-parallel-size-local {{ or .Spec.Parallelism.DataLocal 1 }} \
-                --data-parallel-address $(LWS_LEADER_ADDRESS) \
-                --data-parallel-rpc-port {{ if .Spec.Parallelism.DataRPCPort }}{{ .Spec.Parallelism.DataRPCPort }}{{ else }}5555{{- end }} \
-                --data-parallel-start-rank $START_RANK \
-                --trust-remote-code \
-                --headless \
-                --enable-ssl-refresh \
-                --ssl-certfile \
-                /etc/ssl/certs/tls.crt \
-                --ssl-keyfile \
-                /etc/ssl/certs/tls.key
-            fi
+            START_RANK=0
+            eval "vllm serve \
+              /mnt/models \
+              --served-model-name "{{ .Spec.Model.Name }}" \
+              --port 8001 \
```

Comment:
Shouldn't the port be parameterized?

Reply:
No, why? most users won't need to change it (as this port is also referenced in the routing-sidecar)